.ado/windows-vs-pr.yml

```diff
@@ -24,57 +24,12 @@ stages:
               targetType: filePath
               filePath: .ado/scripts/shouldSkipPRBuild.ps1
 
-  - stage: Build
-    displayName: Build 🔨
-    dependsOn: Setup
-    condition: ne( dependencies.Setup.outputs['checkPayload.shouldSkipPRBuild'], 'True' )
-    jobs:
-      - template: jobs/universal.yml
-        parameters:
-          buildEnvironment: PullRequest
+          - template: templates/yarn-install.yml
```

Comment:
Adding matching steps to properly version stamp builds like in production to ensure we have more similar workflows.